app/lib/methods/handleImageBase64.ts

```diff
@@ -0,0 +1,5 @@
+export const imageBase64RegExp = new RegExp(/^data:image\/[bmp,gif,ico,jpg,png,svg,webp,x\-icon,svg+xml]+;base64,/);
```

Comment:
Is it worth testing the file extension?
Can't we just `/data:image\/[^;]+;base64,/`?